src/Microsoft.AspNetCore.RuntimeStore/Microsoft.AspNetCore.RuntimeStore.csproj

```diff
@@ -17,6 +17,6 @@
         </DepsFiles>
       </ItemGroup>
 
-      <Copy SourceFiles="%(DepsFiles.FullPath)" DestinationFiles="$(DepsOutputPath)\%(DepsFiles.PackageName)\Microsoft.NETCore.App\2.0.0\deps.json" />
+      <Copy SourceFiles="%(DepsFiles.FullPath)" DestinationFiles="$(DepsOutputPath)\%(DepsFiles.PackageName)\Microsoft.NETCore.App\$(RuntimeFrameworkVersion)\deps.json" />
```

Comment:
@glennc also found that we need to add `shared` into this path. Also the file name needs to be <something>.deps.json instead of just deps.json.

so `DestinationFiles="$(DepsOutputPath)\%(DepsFiles.PackageName)shared\\Microsoft.NETCore.App\$(RuntimeFrameworkVersion)\%(DepsFiles.PackageName).deps.json"`